package.json

```diff
@@ -5,7 +5,8 @@
   "dependencies": {
     "react": "^16.8.6",
     "react-dom": "^16.8.6",
-    "react-scripts": "3.0.1"
+    "react-scripts": "3.0.1",
+    "react-sound": "^1.2.0"
```

Comment:
You appear to have not used this dependency. I would recommend not committing dependencies you have not used.